sonar-plugin/src/main/java/io/ecocode/javascript/TypeScriptRulesDefinition.java

```diff
@@ -0,0 +1,41 @@
+package io.ecocode.javascript;
+
+import org.sonar.api.SonarRuntime;
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonarsource.analyzer.commons.RuleMetadataLoader;
+
+import java.util.Collections;
+
+public class TypeScriptRulesDefinition implements RulesDefinition {
+
+    private static final String METADATA_LOCATION = "io/ecocode/javascript/rules/javascript";
```

Comment:
it is perhaps better to put the definitions of the rules in a directory dedicated to the TypeScript language (and not JavaScript)?

```suggestion
    private static final String METADATA_LOCATION = "io/ecocode/rules/ts";
```

Reply:
@jycr This is not as simple as it looks like.. JavaScript rules are also loaded into Typescript language for almost all of them, because they are also applicable to Typescript. Putting the JSON/HTML files in a separate folder would mean duplicating all of them. That's why I've put everything in the folder `javascript`, for simplicity's sake.

So, I propose to put everything in the `io/ecocode/rules/javascript` folder